src/main/java/aquality/selenium/core/utilities/IUtilitiesModule.java

```diff
@@ -0,0 +1,27 @@
+package aquality.selenium.core.utilities;
+
+/**
+ * Provides implementations for utilities module.
+ */
+public interface IUtilitiesModule {
+
+    /**
+     * @return implementation of {@link IElementActionRetrier}.
+     */
+    default Class<? extends IElementActionRetrier> getElementActionRetrierImplementation() {
```

Comment:
leave simple 'getElementActionRetrier' without 'Implementation'

Reply:
I don't agree. This method returns not the ElementActionRetrier but just the Class (type) which implements IElementActionRetrier